### PR TITLE
Add collapsable() resource group

### PR DESCRIPTION
**Pull request by falconlv**
Adds option to toggle resource group

### CODE REVIEW

**Comment by leandrogehlen** [comment]
I'm not sure about it.

The PR does not allow to customize this option.

**Comment by falconlv** [comment]
Hello.

I believe there is no need to customize toggling option on config because my PR just adds a right chevron for those who would like to save space on left side for more commonly used resources than Roles & Permissions. 
By default they are still expanded.

**Comment by leandrogehlen** [comment]
Make sense.
Please, add changelog line

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## v1.2.3 (2023-02-28)
```
Comment (leandrogehlen) [inline CHANGELOG.md:5]: Please remove this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+
+- Added collapsable() for the section
```
Comment (leandrogehlen) [inline CHANGELOG.md:7]: `Menu secition now is collapsable` i think is better

**Comment by falconlv** [comment]
Thank you for merging. This was my first PR ever on Github.

**Pull request merged**